sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubConsumer.cs

```diff
@@ -291,9 +292,20 @@ protected EventHubConsumer()
                                                                           [EnumeratorCancellation] CancellationToken cancellationToken = default)
```

Comment:
nit: not 100% sure about what the convention is, but I guess there should be one more blank space.

Reply:
Yup, I think you're correct.   Adjusted.